libs/barista-components/filter-field/src/filter-field-util.ts

```diff
@@ -371,6 +371,20 @@ export function defaultTagDataForFilterValuesParser(
       )
     : null;
 }
+/** Default function to parse text during the edition of a filed */
+export function defaultEditionDataForFilterValuesParser<T>(
+  filterValues: DtFilterValue[],
+): string {
+  let value: string = '';
+  if (!filterValues?.length) {
+    return value;
+  }
```

Comment:
This was not part of the initial `updateFilterByLabel` but seems like a good addition. You might not need the typing if you already assign it to an empty string. 
```suggestion
  let value: = '';
  if (!filterValues?.length) {
    return value;
  }
```